packages/web/components/patterns/LinkedItemCard.tsx

```diff
@@ -63,13 +81,33 @@ export function GridLinkedItemCard(props: LinkedItemCardProps): JSX.Element {
           overflow: 'clip',
           border: '1px solid $grayBorder',
           boxShadow: '0px 3px 11px rgba(32, 31, 29, 0.04)',
+          position: 'relative',
         }}
         alignment='start'
         distribution='start'
         onClick={() => {
           props.handleAction('showDetail')
         }}
+        className="grid-item-card"
       >
+        <Box css={{
+          position: 'absolute',
+          top: '1px',
+          left: '1px',
+          width: 'calc(100% - 2px)',
+          '& > div': {
```

Comment:
I'm not familiar with applying 100vmax. What does this do? Does it have to be applied here or can it be set directly on the component being targeted?

Reply:
It makes the right edge of the progress bare round, and it is set like that because the progress bare is used in the list item as well, and I preferred to set this styling from the grid card instead of change the progress bar component for that reason, since that change only style related and particular to the grid view only